contents/dataframes_performance.md

```diff
@@ -4,8 +4,240 @@ So far, we didn't think about making our `DataFrames.jl` code **fast**.
 Like everything in Julia, `DataFrames.jl` can be really fast.
 In this section, we will give some performance tips and tricks.
 
-```{=comment}
-- Functions with a shebang versus without (e.g., filter! versus filter)
-- df[!, col] versus df[:, col]
-- other things? Jose probably knows some more
+### Allocations
```

Comment:
```suggestion
### In-place operations 
```

Reply:
The difference between having or not having allocations is a little more than just using a bang !